newroutes/authenticated/sites.js

```diff
@@ -17,6 +21,13 @@ class SitesRouter {
     return res.status(200).json({ siteNames })
   }
 
+  async createSite(req, res) {
+    // TODO: authenticate request from formsg
+    const testParams = {}
+    this.infraService.createSite(testParams)
+    return res.status(200).send("OK")
+  }
+
```

Comment:
Leaving here for reference: the current state of our authentication middleware will not allow submissions from formsg to enter this router (since it requires the appropriate cookies) - we can insert this route back again in future when site creation requests are done through the CMS, but for now, submissions through formsg will need to live in a separate router that bypasses the authentication middleware